__tests__/commands/run.js

```diff
@@ -0,0 +1,39 @@
+/* @flow */
+
+import {run} from '../../src/cli/commands/run.js';
+import * as reporters from '../../src/reporters/index.js';
+import Config from '../../src/config.js';
+
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 60000;
+
+const path = require('path');
+
+const fixturesLoc = path.join(__dirname, '..', 'fixtures', 'run');
+
+async function runRun(
+  flags: Object,
+  args: Array<string>,
+  name: string,
+): Promise<void> {
+  const cwd = path.join(fixturesLoc, name);
+  const reporter = new reporters.NoopReporter();
+  const config = new Config(reporter);
+  await config.init({cwd});
+  return run(config, reporter, flags, args);
+}
+
+test.concurrent('run should run script', (): Promise<void> => {
+  return runRun({}, ['some-script'], 'run-should-run-script');
```

Comment:
I couldn't find a way to do that, but instead I've added expectations for calling of executeLifecycleScript().

I think these tests should only verify that run() can call executeLifecycleScript() properly, then another test can verify if executeLifecycleScript() can do its job properly.